### PR TITLE
Added supported operations for CPX adapter under heading "Sample Appl…

### LINKED ISSUE #1489: [Docs] Update the list of supported operations for the Meshery adapter for Citrix (CPX)
**Current State:**

The operations supported by the adapter for Linkerd are not listed in its [documentation](https://meshery.layer5.io/docs/service-meshes/adapters/cpx).

**Desired State:**

Under the heading `Sample Applications` and include a list of the supported operations, with a small explanation for each. The operations can be found listed [here](https://github.com/layer5io/meshery-cpx/blob/master/cpx/supported_ops.go); (lines 47-124). Look out for the following line: `opType: meshes.OpCategory_SAMPLE_APPLICATION`. This means that the operation listed is a sample application.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by ghost**
…ication"

Signed-off-by: Efereyan Karen Simisola <thedevkaren@gmail.com>

**Description**

This PR fixes #1489

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ X] Yes, I signed my commits.

### CODE REVIEW

File: docs/_adapters/cpx.md [hunk 1]
```diff
@@ -8,33 +8,34 @@ project_status: beta
...
-   
+- [httpbin](https://httpbin.org)
+  - Httpbin is a simple HTTP request and response service.
 - [Istio BookInfo Application](https://github.com/layer5io/istio-service-mesh-workshop/blob/master/lab-2/README.md#what-is-the-bookinfo-application)
-    - This application is a polyglot composition of microservices that are written in different languages. The application displays information about a book, similar to a single catalog entry of an online book store. Displayed on the page is a description of the book, book details (ISBN, number of pages, and so on), and a few book reviews.
+  - This application is a polyglot composition of microservices that are written in different languages. The application displays information about a book, similar to a single catalog entry of an online book store. Displayed on the page is a description of the book, book details (ISBN, number of pages, and so on), and a few book reviews.
+
+### Lifecycle management
+
+The {{page.name}} can install **{{page.version}}** of the {{page.name}} service mesh. The SMI adapter for Kuma can also be installed using Meshery.
```
Comment (leecalcote) [inline docs/_adapters/cpx.md:38]: Thanks!

1. We'll want to remove any references to Kuma in this doc.
1. Currently, the Meshery adapter for CPX does not support running SMI conformance tests.

File: docs/_adapters/cpx.md [hunk 2]
```diff
@@ -8,33 +8,34 @@ project_status: beta
...
+| {{page.title}} | [{{ page.project_status }}]({{ page.github_link }}) |       {{page.version}}        |
 
 ### Features
-1. Lifecycle Management of Citrix 
-2. Lifecycle Management of Sample Applications
-
 
-### Lifecycle management
-The {{page.name}} can install **{{page.version}}** of the {{page.name}} service mesh. The SMI adapter for Kuma can also be installed using Meshery.
+1. Lifecycle Management of Citrix
```
Comment (leecalcote) [inline docs/_adapters/cpx.md:20]: Please update to "Lifecycle Management of Citrix Service Mesh", if you would.

Comment (leecalcote) [reply docs/_adapters/cpx.md:20]: Actually, even better to update to "Lifecycle Management of {{ page.name }}"

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/_adapters/cpx.md (hunk 2 above)
Comment (KarenEfereyan) [reply docs/_adapters/cpx.md:20]: On it

**Comment by KarenEfereyan** [comment]
@leecalcote  done

File: docs/_adapters/cpx.md (hunk 2 above)
Comment (KarenEfereyan) [reply docs/_adapters/cpx.md:20]: @leecalcote  I'm done. Could you review the changes?

**Review by leecalcote — APPROVED** [review]
Good to go. 👍 Thank you, @KarenEfereyan

**Pull request merged**